core/src/main/java/com/oath/oak/Rebalancer.java

```diff
@@ -16,14 +16,16 @@
 
     /*-------------- Constants --------------*/
 
-    private final int rebalanceSize;
-    private final double maxAfterMergePart;
-    private final double lowThreshold;
-    private final double appendThreshold;
+    private final int rebalanceSize = 2;
```

Comment:
should these parameters be static and with UPPER CASE?

Reply:
Will change